Octokit/Models/Response/Account.cs

```diff
@@ -67,6 +67,11 @@ protected Account(string avatarUrl, string bio, string blog, int collaborators,
         /// </summary>
         public DateTimeOffset CreatedAt { get; protected set; }
 
+        /// <summary>
+        /// Date the account was updated.
+        /// </summary>
+        public DateTimeOffset UpdatedAt { get; protected set; }
```

Comment:
Does a response to `GetOrganization` populate this field (the API docs don't show it but I'm not able to test a real response from my phone right now). If it's not on `Organization` then this field would need to be moved to `User` rather than the `Account` base class